File: Programacao/1012.c
#include<stdio.h>
#include<math.h>
#define pi 3.14159

int main(){
	
	int i;
	double triangulo, circulo,trapezio,quadrado, retangulo, x[3];
	
	for(i=0;i<3;i++){
	scanf("%lf",&x[i]);
	}
	
	triangulo=(x[0]*x[2])/2;
	circulo= pi* pow(x[2],2);
	trapezio=((x[0]+x[1])*x[2])/2;
	quadrado=pow(x[1],2);
	retangulo=x[0]*x[1];
	
	printf("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f\n",triangulo,circulo,trapezio,quadrado,retangulo);
	
	return 0;
}

File: Java/1012.java
import java.util.Scanner;

class Main{

    public static void main(String[] args){
        Scanner entrada = new Scanner(System.in);

        double a, b, c, triangulo, circulo, trapezio, quadrado,retangulo, pi = 3.14159;
        
        a = entrada.nextDouble();
        b = entrada.nextDouble();
        c = entrada.nextDouble(); 

        triangulo = (a*c)/2;
        circulo = pi* Math.pow(c,2);
        trapezio = ((a+b)*c)/2;
        quadrado = b*b;
        retangulo = a*b;

        System.out.printf("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f\n",triangulo,circulo, trapezio, quadrado, retangulo);
    }
}